tests/foreman/cli/test_computeresource_libvirt.py

```diff
@@ -422,3 +425,82 @@ def test_positive_update_console_password(libvirt_url, set_console_password):
     cr_name = gen_string('utf8')
     ComputeResource.create({'name': cr_name, 'provider': 'Libvirt', 'url': gen_url()})
     ComputeResource.update({'name': cr_name, 'set-console-password': set_console_password})
+
+
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.tier3
+@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete=True'], indirect=True)
+def test_positive_provision_end_to_end(
+    request,
+    setting_update,
+    module_libvirt_provisioning_sat,
+    module_sca_manifest_org,
+    module_location,
+    provisioning_hostgroup,
+):
+    """Provision a host on Libvirt compute resource with the help of hostgroup.
+
+    :id: b003faa9-2810-4176-94d2-ea84bed248ec
+
+    :setup: Hostgroup and provisioning setup like domain, subnet etc.
+
+    :steps:
+        1. Create a Libvirt compute resource.
+        2. Create a host on Libvirt compute resource using the Hostgroup
+        3. Use compute-attributes parameter to specify key-value parameters
+           regarding the virtual machine.
+        4. Provision the host.
+
+    :expectedresults: Host should be provisioned with host group
+    """
+    sat = module_libvirt_provisioning_sat.sat
+    cr_name = gen_string('alpha')
+    hostname = gen_string('alpha').lower()
+    libvirt_cr = sat.cli.ComputeResource.create(
+        {
+            'name': cr_name,
+            'provider': FOREMAN_PROVIDERS['libvirt'],
+            'url': LIBVIRT_URL,
+            'organizations': module_sca_manifest_org.name,
+            'locations': module_location.name,
+        }
+    )
+    assert libvirt_cr['name'] == cr_name
+    host = sat.cli.Host.create(
+        {
+            'name': hostname,
+            'location': module_location.name,
+            'organization': module_sca_manifest_org.name,
+            'hostgroup': provisioning_hostgroup.name,
+            'compute-resource-id': libvirt_cr['id'],
+            'ip': None,
+            'mac': None,
```

Comment:
Non blocking: Is passing `None` here necessary, won't the default be `None` ?

Reply:
yes, it will be default to None, but I wanted to be explicit here, as I'm removing existing test_positive_create_using_libvirt_without_mac test